### DISCUSSION TITLE
Custom Json Config file import

### DISCUSSION THREAD

**Discussion by Learner-Xdot** [v2rayA/v2rayA#1479] [Ideas]
Hello Guys,

       I don't know if it's my require or not , If it's not I hope it'll be great ideas to u guys, because i don't see any import custom json config file in v2raya i'm not familiar that much tho i was googling about that and searched on youtube too but i didn't see nothing about it so suggestion to add import custom json file for v2raya . Isn't good ?

**Comment by arminmacx** [comment]
well all of us i think need that or at least update to v2raya so it can handle new protocol like httpupgrade or splithttp and ... but for now i dont know what developer think or even working on  to upgrade it or not. and its limited to only chinese for direct dns as i need to change it for my own situation but for now its not allow us
